src/styles/star.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background: url(../images/star.svg) no-repeat center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--1 &__star:nth-child(-n + 1),
+  &--2 &__star:nth-child(-n + 2),
+  &--3 &__star:nth-child(-n + 3),
+  &--4 &__star:nth-child(-n + 4),
+  &--5 &__star:nth-child(-n + 5) {
+    background-image: url(../images/star-active.svg);
```

Comment:
<img width="649" alt="Screenshot 2023-08-27 at 16 03 23" src="https://github.com/mate-academy/layout_catalog/assets/135232001/169243bb-15ef-4791-9047-9724915e9cdc">
Consider following the checklist to improve your code